index.html

```diff
@@ -0,0 +1,324 @@
+            <li><a href="index.html">Ruby</a></li>
+          </ul>
+        </div>
+        <div class="seeproject none">
+          <a href="index.html">See project</a>
+        </div>
+      </section>
+      <section class="project5">
+        <div class="text1">
+          <h2>Proffessional Art Printing Data</h2>
+        </div>
+        <div class="text2">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has ben the industry's standard dummy text ever
+            since the 1500s
+          </p>
+        </div>
+        <div class="projlang">
+          <ul>
+            <li><a href="index.html">HTML</a></li>
+            <li><a href="index.html">Bootstrap</a></li>
+            <li><a href="index.html">Ruby</a></li>
+          </ul>
+        </div>
+        <div class="seeproject none">
+          <a href="index.html">See project</a>
+        </div>
+      </section>
+    </div>
+      <section class="about-me">
+        <h1>About me</h1>
+        <div class="details">
+          <p>
+            Hello, i'm a software developer, I can help you build a
+            proffessional website, feature. Look through some of my work and
+            experience and if you are pleased with my work, contact me.
+          </p>
+        </div>
+        <div class="resume">
+          <a class="res" href="#">Get my resume</a>
+        </div>
+        <div class="about-me-img">
+          <img src="images/images-self/line.png" />
+        </div>
+        <div class="experience">
+          <ul>
+            <li class="lang">
+              <img src="images/images-self/Rectangle 55-language.png" />
+              <h4>Language</h4>
+            </li>
+            <li><a href="#">Java Script</a></li>
+            <li><a href="#">Ruby</a></li>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">CSS</a></li>
+          </ul>
+          <ul>
+            <li class="frameworks">
+              <img src="images/images-self/Rectangle 56-framework.png" />
+              <h4>Frameworks</h4>
+            </li>
+            <li><a href="#">React.js</a></li>
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">Rspec</a></li>
+            <li><a href="#">Capybara</a></li>
+            <li class="Skills">
+            </ul>
+            <ul>
+              <li>
+              <img src="images/images-self/Rectangle-skills.png" />
+              <h4>Skills</h4>
+            </li>
+            <li><a href="#">Database management</a></li>
+            <li><a href="#">Version control</a></li>
+            <li><a href="#">CLI</a></li>
+            <li><a href="#">Webdevelopment</a></li>
+          </ul>
+        </div>
+      </section>
+      <footer class="form-area">
+        <div class="creator">
+          <h2>I'm always interested in hearing about new projects, so if you would like to chat 
+            please get in touch.
```

Comment:
- [ ] Greate job :+1: ensuring that the title of your contact me section is aligned with the corresponding fields as requested by the previous code reviewer. But while making the changes, the text here should be aligned left not center as it is in the Figma design :point_down: 
  ### Your Design
  ![Screenshot from 2023-05-06 15-13-58](https://user-images.githubusercontent.com/92303891/236632719-2be4e937-ea96-4e6c-99c3-ed15b809e6b7.png)
  
  ### Figma design 
  ![image](https://user-images.githubusercontent.com/92303891/236632749-d3a9a661-96df-450e-af4a-13082abba627.png)

- [ OPTIONAL ] Why don't you add the last name field to the contact field of your project as it is in the Figma design